src/ast/codegen_llvm.cpp

```diff
@@ -1679,12 +1679,13 @@ void CodegenLLVM::createLinearFunction()
   b_.CreateCondBr(cmp1, gt_max, le_max);
 
   b_.SetInsertPoint(gt_max);
-  Value *div = b_.CreateSDiv(b_.CreateSub(b_.CreateLoad(max_alloc), b_.CreateLoad(min_alloc)), b_.CreateLoad(step_alloc));
+  // div(max - min), step
```

Comment:
Shouldn't this be `div((max-min), step)`? At any rate, this is already documented above. I think we can delete this.

Reply:
Woops, removed it :)